api/src/main/java/org/openmrs/module/m2sysbiometrics/client/M2SysV105Client.java

```
@@ -81,6 +81,7 @@ public EnrollmentResult enroll(BiometricSubject subjectId, String fingerprintXml
         if (!fingerScanStatus.isRegisteredLocally()) {
             if (fingerScanStatus.isRegisteredNationally()) {
                 registrationService.fetchFromMpiByNationalFpId(fingerScanStatus.getNationalBiometricSubject(), capture);
+                registrationService.importCcd(fingerScanStatus.getNationalBiometricSubject());
```

Can you catch all exceptions here, log and proceed on the failure